app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -87,3 +87,12 @@
     border-color: govuk-colour("black");
   }
 }
+
+// new colour for DSIT for use until government-frontend has it's next deploy
```

Comment:
Curious to understand this comment - what's the relationship between this and the next deployment of `government-frontend`?

Reply:
Whoops that should say govuk-frontend

Reply:
Ah that makes sense! Thanks.